### DISCUSSION TITLE
Motion Recognition based on Edge Impulse | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#351] [Q&A]
# Motion Recognition based on Edge Impulse | Seeed Studio Wiki

Motion Recognition based on Edge Impulse

https://wiki.seeedstudio.com/XIAOEI/

**Comment by Dom-888** [comment]
Do you remember what size screws you used for this project?

**Reply by limengdu** [reply]
M2 self-tapping screws

**Reply by Dom-888** [reply]
Thanks a mil
